//
// Academic License - for use in teaching, academic research, and meeting
// course requirements at degree granting institutions only.  Not for
// government, commercial, or other organizational use.
//
// acefg_T_AV.h
//
// Code generation for function 'acefg_T_AV'
//

#ifndef ACEFG_T_AV_H
#define ACEFG_T_AV_H

// Include files
#include "rtwtypes.h"
#include <cstddef>
#include <cstdlib>

// Function Declarations
void acefg_T_AV(double V_init, double A_init, double V_wayp, double A_wayp,
                double A_min, double J_max, double J_min, double T,
                creal_T t[14]);

#endif
// End of code generation (acefg_T_AV.h)
